### ISSUE TITLE
Chart click bug

### ISSUE THREAD

**Issue by Mikanoshi** [amcharts/amcharts5#398]
Adding `chart.events.on("click", ...)` makes some elements (zoom out button, scrollbar grips) react to mouse hover incorrectly:
https://codepen.io/Mikanoshi/pen/ExQYXvb

**Comment by martynasma** [comment]
I tried your Codepen, but was not able to locate any anomalies with button hovers. Any tips on how to reproduce it?

**Comment by Mikanoshi** [comment]
Zoom out button has a horizontal line in the center, grips have 2 vertical lines, these regions act like there is no hover on them, and they are also click through, like someone has cut holes in those buttons :)

**Comment by martynasma** [comment]
Thanks.

You can fix that right now by adding `interactiveChildren: false` on your chart:

```JavaScript
var chart = root.container.children.push(am5xy.XYChart.new(root, {
  panX: true,
  panY: true,
  wheelX: "panX",
  wheelY: "zoomX",
  pinchZoomX:true,
  interactiveChildren: false
}));
```

Or wait until next version, where we will do that by default.

**Comment by martynasma** [comment]
Fixed in [5.2.1](https://github.com/amcharts/amcharts5/releases/tag/5.2.1).

## [5.2.1] - 2022-05-03

### Added
- `centerMapOnZoomOut` setting added to `MapChart` (default: `true`). Means the map will center itself (or go to `homeGeoPoint` if set) when fully zoomed out.
- `getGroupInterval(duration)` method added to `DateAxis`. Returns a time interval axis would group data to for a specified duration.
- `getIntervalMax(interval)` and `getIntervalMin(interval)` methods added to `DateAxis`. These methods return `min` and `max` of a specified time interval.

### Changed
- Axis fills on a `StockChart` are now not visible by default.
- All `Chart` elements now have `interactiveChildren: true` set by default, so that adding events on a chart does not automatically make all elements in it interactive.
- Zooming out `MapChart` using mouse wheel will now auto-center the map by default. Set `centerMapOnZoomOut: false` to bring back old behavior.

### Fixed
- Changing `inside` and `opposite` of `AxisRendererX` or `AxisRendererY` after the axis was already initialized was not working properly.
- `StockChart`'s' period selection buttons were not zooming to exact dates when data grouping of an X-axis was turned on.
- `StockChart`'s' series type switcher was not switching series properly when data grouping was enabled.
- `StockChart`'s' series type switcher was not switching series properly when comparing mode was on.
- If a `StockChart` was in percent scale mode, adding an indicator which was not on a separate panel was resulting in the indicator using wrong scale.
- Fixed week number calculation (again).
- Fixed CSS of `StockToolbar` for FireFox.
- Pinch-zoom of `MapChart` was ignoring `minZoomLevel` and `maxZoomLevel` settings.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
